### PR TITLE
Allow multiple --env arguments to be passed to p2-exec

### PR DESCRIPTION
**Pull request by mpuncel**
This will allow us to provide environment variables from multiple
directories to a process. The immediate motivation is to combine
pod-wide environment variables with launchable-specific environment
variables.

### CODE REVIEW

**Comment by mpuncel** [comment]
Should this have any protection against conflicting environment variables being set in the various directories?

File: bin/p2-exec/main.go [hunk 1]
```diff
@@ -39,8 +39,8 @@ func main() {
 		os.Clearenv()
 	}
 
-	if *envDir != "" {
-		err := loadEnvDir(*envDir)
+	for _, dir := range *envDir {
```
Comment (anthonybishopric) [inline bin/p2-exec/main.go]: Small request: It'd be nice if the arg description said in what order the env directories are processed - thinking about situations where a launchable needs to override a global value or something.

**Comment by anthonybishopric** [comment]
I think it's desirable to have overlapping entries, it just needs to be clearly stated how the invocation would work.

**Comment by mpuncel** [comment]
updated to show in help text that the directory appearing last will win

**Comment by anthonybishopric** [comment]
:+1:

**Pull request merged**